benchmark/Makefile

```diff
@@ -1,28 +1,20 @@
-GOPATH    ?= $(shell go env GOPATH)
-GOIMPORTS ?= $(GOPATH)/bin/goimports
-DEP       ?= $(GOPATH)/bin/dep
+GOPATH      ?= $(shell go env GOPATH)
+GOIMPORTS   ?= $(GOPATH)/bin/goimports
+GO111MODULE ?= on
+export GO111MODULE
 
 .PHONY: all
-all: deps build
-
-.PHONY: deps
-deps: $(DEP)
-	@echo ">> fetching dependencies"
-	@$(DEP) ensure
+all: build
```

Comment:
can we ensure in here that we also have the `check-go-mod` that will fail CI

Reply:
I don't want to prevent the build because you do not have commited your changes.

I've add a step in the circle-ci runbook which will run `make check-go-mod` and fail if you forgot to tidy mod.

Reply:
Done. I've added the checks to all targets which use go and thus could require the tools.

Reply:
If we add a `go mod tidy` to the `make` it will ensure the modules are tidy *before* you commit. Engineers should not be committing without running `make`.